specs/wp-notifications-spec.js

```diff
@@ -98,7 +98,8 @@ test.describe( 'Notifications: (' + screenSize + ') @parallel', function() {
 					test.it( 'Can delete the comment', function() {
```

Comment:
As long as you're in here can you update this step description to something like "Can delete the comment (and wait for UNDO grace period)" so there's some indication for anyone watching why this step is taking so long?